skype/skype-ps/skype/Set-CsTeamsCallHoldPolicy.md

```diff
@@ -0,0 +1,220 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-help.xml
+online version: https://docs.microsoft.com/powershell/module/skype/set-CsTeamsCallHoldPolicy
+Module Name: MicrosoftTeams
+online version:
+schema: 2.0.0
+title: Set-CsTeamsCallHoldPolicy
+manager: abnair
+author: joelhmarquez
+ms.author: jomarque
+ms.reviewer:
+---
+
+# Set-CsTeamsCallHoldPolicy
+
+## SYNOPSIS
+Modifies an existing Teams call hold policy in your tenant.
+The Teams call hold policy is used to customize the call hold experience for Teams clients.
+
+## SYNTAX
+
+### Identity (Default)
+```
+Set-CsTeamsCallHoldPolicy [-Tenant <System.Guid>] [-Description <String>] [-AudioFileId <String>]
+ [[-Identity] <XdsIdentity>] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### Instance
+```
+Set-CsTeamsCallHoldPolicy [-Tenant <System.Guid>] [-Description <String>] [-AudioFileId <String>]
+ [-Instance <PSObject>] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Teams call hold policies are used to customize the call hold experience for teams clients.
+When Microsoft Teams users participate in calls, they have the ability to hold a call and have the other entity in the call listen to an audio file during the duration of the hold.
+Assinging a teams call hold policy to a user sets an audio file to be played during the duration of the hold.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-TeamsCallHoldPolicy -Identity "ContosoPartnerTeamsCallHoldPolicy" -AudioFileID "c65233-ac2a27-98701b-123ccc" 
+```
+
+The command shown in Example 1 modifies an existing per-user Teams call hold policy with the Identity ContosoPartnerTeamsCallHoldPolicy.
+This policy is re-assigned the audio file id to be used to: c65233-ac2a27-98701b-123ccc, which is the id referencing an audio file that was uploaded using the Import-CsOnlineAudioFile cmdlet.
+
+Any Microsoft Teams users who are assigned this policy will have their call holds customized such that the user being held will hear the audio file specified by AudioFileId.
+
+### Example 2
+```powershell
+PS C:\> Set-TeamsCallHoldPolicy -Identity "ContosoPartnerTeamsCallHoldPolicy" -Description "country music" 
+```
+
+The command shown in Example 2 modifies an existing per-user Teams call hold policy with the Identity ContosoPartnerTeamsCallHoldPolicy.
+This policy is re-assigned the description from its existing value to "country music".
+
+Any Microsoft Teams users who are assigned this policy will have their call holds customized such that the user being held will hear the audio file specified by AudioFileId.

```

Comment:
```suggestion

```

Reply:
I think this sentence was copied and pasted from example 1, but it's not applicable here.